cross/tcl/Makefile

```diff
@@ -1,36 +1,92 @@
-PKG_NAME = tcl
-PKG_VERS = 8.6.6
-PKG_EXT = tar.gz
-PKG_DIST_NAME = $(PKG_NAME)$(PKG_VERS)-src.$(PKG_EXT)
-PKG_DIST_SITE = http://prdownloads.sourceforge.net/tcl/
-PKG_DIR = $(PKG_NAME)$(PKG_VERS)/unix
+PKG_NAME	= tcl
+PKG_VERS	= 8.6.6
+PKG_EXT		= tar.gz
+PKG_DIST_NAME	= $(PKG_NAME)$(PKG_VERS)-src.$(PKG_EXT)
+PKG_DIST_SITE	= http://prdownloads.sourceforge.net/tcl/
+PKG_DIR		= $(PKG_NAME)$(PKG_VERS)/unix
 
-DEPENDS =
-BUILD_DEPENDS = cross/zlib
+DEPENDS		=
+BUILD_DEPENDS	= cross/zlib
 
-HOMEPAGE = http://www.tcl.tk/
-COMMENT  = Tcl (Tool Command Language) is a very powerful but easy to learn dynamic programming language, suitable for a very wide range of uses, including web and desktop applications, networking, administration, testing and many more. Open source and business-friendly, Tcl is a mature yet evolving language that is truly cross platform, easily deployed and highly extensible.
-LICENSE  = http://tcl.tk/software/tcltk/license.html
+HOMEPAGE	= http://www.tcl.tk/
+COMMENT		= Tcl (Tool Command Language) is a very powerful but easy to learn dynamic programming language, suitable for a very wide range of uses, including web and desktop applications, networking, administration, testing and many more. Open source and business-friendly, Tcl is a mature yet evolving language that is truly cross platform, easily deployed and highly extensible.
+LICENSE		= http://tcl.tk/software/tcltk/license.html
 
-GNU_CONFIGURE = 1
+GNU_CONFIGURE	= 1
 
-# Prevent re-declaration of fixstrtod.o
-# export ac_cv_func_strtod=yes
-# export tcl_cv_strtod_buggy=1
+export ac_cv_path_tclsh=/usr/local/bin/tclsh
 
 # Enable 64-bit support in Tcl for 64-bit architectures
 ifneq ($(findstring $(ARCH), avoton braswell bromolow cedarview x64 x86),)
-	CONFIGURE_ARGS = tcl_cv_strtod_buggy=1 --prefix=$(STAGING_INSTALL_PREFIX) --enable-64bit
+	CONFIGURE_ARGS = tcl_cv_strtod_buggy=1 --prefix=$(TCL_PACKAGE_PATH) --enable-64bit
 else
-	CONFIGURE_ARGS = tcl_cv_strtod_buggy=1 --prefix=$(STAGING_INSTALL_PREFIX)
+	CONFIGURE_ARGS = tcl_cv_strtod_buggy=1 --prefix=$(TCL_PACKAGE_PATH)
 endif
 
-POST_INSTALL_TARGET = myPostInstall
+TCL_PACKAGE_PATH	= /var/packages/tcl/target
+
+COMPILE_TARGET		= myCompile
```

Comment:
The override here (and all the code on L33 and further) doesn't make sense imo. Please check if the following approach works:
- Set `CONFIGURE_ARGS= tcl_cv_strtod_buggy=1 --enable-64bit` on L21, and `CONFIGURE_ARGS = tcl_cv_strtod_buggy=1` on L23
(There's probably a better way to structure the code, but let's make sure this works before we get to that)
- Drop the override for `COMPILE_TARGET` completely